lib/incoming_form.js

```diff
@@ -26,7 +26,7 @@ function IncomingForm(opts) {
   this.maxFields = opts.maxFields || 1000;
   this.maxFieldsSize = opts.maxFieldsSize || 2 * 1024 * 1024;
   this.keepExtensions = opts.keepExtensions || false;
-  this.uploadDir = opts.uploadDir || os.tmpDir();
+  this.uploadDir = opts.uploadDir || os.tmpdir();
```

Comment:
@gswalden I'm pretty sure that Node.js 0.8 support can and _should_ be dropped. I.e. not break on a purpose, but don't care about it specifically, so this doesn't need code complication.